invoicer_project/static/js/clients_list_request.js

```diff
@@ -92,33 +94,32 @@ async function addElementsDynamically() {
 function addDeleteButtonListeners() {
     const deleteButtons = document.querySelectorAll('.delete-client');
     deleteButtons.forEach(span => {
-        span.addEventListener('click', () => {
-            let clientId = span.dataset.clientId;
-
-            const requestOptions = {
-                method: 'DELETE',
-                redirect: 'follow',
-                headers: {
-                    'Authorization': `Bearer ${window.localStorage.getItem('accessToken')}`
-                }
-            };
-
-            fetch("http://127.0.0.1:8000/clients/client/" + clientId.toString(), requestOptions)
-                .then(response => {
-                    if (response.ok) {
-                        location.reload();
-                    } else {
-                        console.error('Error deleting client:', response.statusText);
+        span.addEventListener('click', async () => {
+            try {
+                let clientId = span.dataset.clientId;
+                const requestOptions = {
+                    method: 'DELETE',
+                    redirect: 'follow',
+                    headers: {
+                        'Authorization': `Bearer ${window.localStorage.getItem('accessToken')}`
                     }
-                })
-                .catch(error => {
-                    console.error('Error deleting client:', error);
-                });
+                };
+                const response = await fetch(`http://127.0.0.1:8000/clients/client/${clientId.toString()}`, requestOptions);
```

Comment:
please, create global constant for host `http://127.0.0.1:8000/`

Reply:
Added